mods/tnt/init.lua

```diff
@@ -404,7 +404,7 @@ minetest.register_node("tnt:gunpowder", {
 		type = "fixed",
 		fixed = {-1/2, -1/2, -1/2, 1/2, -1/2+1/16, 1/2},
 	},
-	groups = {dig_immediate = 2, attached_node = 1, connect_to_raillike = minetest.raillike_group("gunpowder")},
+	groups = {dig_immediate = 2, attached_node = 1, flammable = 5, connect_to_raillike = minetest.raillike_group("gunpowder")},
```

Comment:
I'm not sure what do you mean.
It does change the neighbour gunpowder to 'gunpowder burning', that was the point of making it flammable, the {{on_burn}} gets triggered and the gunpowder turns into gunpowder burning which then turns its neighbors.

But I think I'm gonna remove this and leave it for another PR, since I'm gonna replace the {{on_burn}} with {{on_ignite}} on this one.